### DISCUSSION TITLE
Is it possible to send email notifications without using an SMTP server?

### DISCUSSION THREAD

**Discussion by serberci** [globaleaks/globaleaks-whistleblowing-software#4553] [Q&A] [answered]
Is there anyway to send notification e-mails without a SMTP server behind? I did not see any configuration to do it.

How can i edit the sender from e-mails? To use a noreply@company.com, for example.

Thanks!

**Comment by evilaliv3** [comment, marked as answer]
Dear @serberci ,

The current implementation requires a properly configured SMTP server to implement a reliable notification and avoid spam.

While technically possible, we have not considered sending email directly from the application because this would require end-users advanced knowledge to setup DNS records in relation to DKIM and SPF. Did you consider this for example while proposing this feature ?

**Comment by serberci** [comment]
Hi @evilaliv3, First of all, thanks a lot for your fast answer and considerations. The problem that i got while using the SMTP server configuration is purely restricted by company issues, the only way that i could use the notification system it's by sending the email as "noreply@company.com", because the company are not allowed to create an email account exclusivity for applications (for security reasons, do not have more explanation or information about that...).

So, they proposed the following, sending the email to the SMTP server as "noreply@company.com", for avoiding SPAM. Is that possible, or the mail sender must be "notifications@globaleaks.org"?

Thanks a lot, great work with this application.

**Comment by evilaliv3** [comment]
Using our default credentials you could only send email as notifications@globaleaks.org

This is due to the fact that our mail server is optimized for anti-spam, apply DKIM signatures and is rigorously hardened.

**Comment by serberci** [comment]
Oh! Okey, i'll try to use our own server. Thanks a lot!
